blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -1,93 +1,24 @@
 import React, { useEffect, useState } from "react";
 import PropTypes from "@arc-fusion/prop-types";
-import styled from "styled-components";
 import { useContent } from "fusion:content";
 import { useFusionContext } from "fusion:context";
 import getProperties from "fusion:properties";
 import getThemeStyle from "fusion:themes";
 import getTranslatedPhrases from "fusion:intl";
 import FocusTrap from "focus-trap-react";
+import { Button, Stack } from "@wpmedia/arc-themes-components";
 import { generateNavComponentPropTypes } from "./nav-helper";
 import SectionNav from "./_children/section-nav";
 import NavLogo from "./_children/nav-logo";
 import HorizontalLinksBar from "./_children/horizontal-links/default";
 import NavSection from "./_children/nav-section";
 import MenuWidgets from "./_children/menu-widgets";
-// shares styles with header nav block
-// can modify styles in shared styles block
-import "@wpmedia/shared-styles/scss/_header-nav.scss";
-
-/* Global Constants */
-// Since these values are used to coordinate multiple components, I thought I'd make them variables
-// so we could just change the vars instead of multiple CSS values
-const standardNavHeight = 56;
-const navZIdx = 9;
-const sectionZIdx = navZIdx - 1;
-
-// max-width 100% fills the available space on smaller viewports
-// max-width 240px is the biggest the icon will grow to on larger viewports
-/* Styled Components */
-const StyledNav = styled.nav`
-	align-items: center;
-	width: 100%;
-	position: sticky;
-	top: 0;
-	margin-bottom: 0;
-	z-index: 1;
-
-	.news-theme-navigation-bar {
-		@media screen and (max-width: ${(props) => props.breakpoint}px) {
-			height: ${standardNavHeight}px;
-		}
-		@media screen and (min-width: ${(props) => props.breakpoint}px) {
-			height: ${(props) => (props.scrolled ? standardNavHeight : props.navHeight)}px;
-		}
-		background-color: ${(props) => props.navBarBackground};
-		transition: 0.5s;
-		z-index: ${navZIdx};
-	}
-
-	.nav-logo {
-		img {
-			height: auto;
-			max-width: 240px;
```

Comment:
image may be pending resizer. in either case, we want to match the same image logic as before

Reply:
Its matching up in storybook pretty good for now.  Definitely waiting for resizer, if I understand this correctly.

Reply:
huh ok. maybe I'm missing something with the image resizing https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=62cec65fd7171d035fa6ec58
<img width="620" alt="Screen Shot 2022-07-15 at 10 14 36" src="https://user-images.githubusercontent.com/5950956/179193381-01144258-b167-47d6-b292-a342d2ad25bf.png">